webapp/accounts/static/accounts/style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
```

Comment:
Any reason this is still here?
Global CSS changes should go in the static files' dir of the `geek_text` app.

Reply:
Yes, I will remove that. There is no reason for that to be there